docs/generate_docs.py

```diff
@@ -0,0 +1,51 @@
+# generate_docs.py
+def generate_markdown_content():
+    # Generate your Markdown content here
+    # Claire Holmes August 2023
+    # prints the NSD images and their HED annotations to a markdown file for a Read the Docs page 
+
+    import pandas
+    import datetime
+    from mdutils.mdutils import MdUtils
+
+    # load tsv of image identifiers and annotations
+    shared1000 = pandas.read_table('shared1000_HED.tsv')
+
+    # start markdown file
+    mdFile = MdUtils(file_name='nsd_images_annotations', title='Images and Annotations', author= 'Claire Holmes')
+    mdFile.new_line(str(datetime.datetime.utcnow()) + '\n')
```

Comment:
Few suggestions here:
1. Add text before timestamp, perhaps 'Last update:'.
2. Format timestamp. For example: `datetime.datetime.utcnow().strftime('%B %d %Y - %H:%M:%S')`
3. Move to end of text before table.